habana/pytorch/buildspec.yml

```diff
@@ -47,15 +47,15 @@ images:
         VAR_IN_DOCKERFILE: PT_TRAINING_URL
```

Comment:
@waytrue17 Can you please update URI above to aws 1.9.1 + habana pytorch framework

Reply:
Thank you for the pointers. I'll be updating them when framework binaries are ready @omrialmog

Reply:
This configuration for specifying the binary is not needed. We can directly specify the binary in the dockerfile like [PT1.9 dockerfile](https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.9/py3/Dockerfile.cpu#L31)

Lets remove the `pytorch_binary` configuration from this file